src/frame/js/actions/integrations.js

```diff
@@ -250,11 +279,17 @@ export function linkSMSChannel(userId, data) {
     };
 }
 
-export function unlinkSMSChannel(userId, type) {
+export function unlinkSMSChannel(type) {
     return (dispatch, getState) => {
-        return core(getState()).appUsers.unlinkChannel(userId, type)
+        const {config: {appId}, user: {_id, clients, pendingClients}} = getState();
+        const client = clients.find((client) => client.platform === type);
+
+        if (!client) {
+            return Promise.resolve();
+        }
+
+        return dispatch(http('DELETE', `/client/apps/${appId}/appusers/${_id}/clients/${client._id}`))
```

Comment:
clients are accessed by their `.id` not `._id`. In fact I don't think we return `._id` in the API at all for a client. We might want to discuss this though